includes/Admin/ReviewNotice.php

```diff
@@ -0,0 +1,153 @@
+<?php
+
+namespace WeDevs\WeMail\Admin;
+
+/**
+ * Shiw admin notice.
+ */
+class ReviewNotice {
+
+    private static $instance;
+    public $time_based_review;
+    public $day_count;
+    public $campaign_count;
+
+    public static function instance() {         if ( ! isset( self::$instance ) ) {
+            self::$instance = new ReviewNotice();
+            self::$instance->boot();
+	}
+
+        return self::$instance;
+    }
+
+    /**
+     * Boot the notice
+     */
+    private function boot() {
+        add_action( 'admin_enqueue_scripts', [ $this, 'enqueue_assets' ] );
+    }
+
+    public function enqueue_assets() {
+        wp_register_style( 'wemail-review-notice-style', WEMAIL_ASSETS . '/css/reviewNotice.css', false, filemtime( WEMAIL_PATH . '/assets/css/reviewNotice.css' ) );
```

Comment:
`wp_register_style` and `wp_register_script` 4th argument expect version number. To get the weMail plugin version number just use `WEMAIL_VERSION` constant.